hyperion/runner.py

```diff
@@ -291,7 +291,13 @@ def main() -> None:
     root_logger = logging.getLogger()
     stream_formatter = ColorFormatter()
     for handler in root_logger.handlers:
-        handler.setFormatter(stream_formatter)
+        if isinstance(handler, logging.StreamHandler):
+            handler.setFormatter(stream_formatter)
```

Comment:
move setFormatter out of the condition. FileHandlers should also use the color formatting (stream_formatter might not be the ideal name, we could change it to color_formatter)